Gemfile

```diff
@@ -4,8 +4,9 @@ gemspec
 gem 'json', '~> 1.8.3'
 gem 'rest-client', '~> 1.8.0'
 gem 'activesupport', '~> 4.2.3'
+gem 'pmckee11-redis-lock', require: 'redis-lock'
```

Comment:
I put it here due to the `require` bit, couldn't figure out a way to do that in the Gemspec